test/functional-tests/app.spec.js

```diff
@@ -1,8 +1,18 @@
-describe('Page title', function () {
-  it('should be Paratii', function () {
-    browser.url('http://localhost:8080')
-    browser.waitUntil(() => {
-      return browser.getTitle() === 'Paratii'
+describe('Portal application', () => {
+  describe('Page title', function () {
+    it('should be Paratii', () => {
+      browser.url('http://localhost:8080')
+      browser.waitUntil(() => {
+        return browser.getTitle() === 'Paratii'
+      })
+    })
+  })
+  describe('Nav bar', () => {
+    it("should render the user's PTI balance", () => {
+      browser.url('http://localhost:8080')
+      browser.waitUntil(
+        () => browser.getText('[data-test-id="nav-pti-balance"]') === '0 PTI'
```

Comment:
@jellegerbrandy suggestions on how to set up a test where the user has a balance other than `0`?